torrent/api/src/commonMain/kotlin/api/files/TorrentFileEntry.kt

```diff
@@ -183,9 +194,9 @@ abstract class AbstractTorrentFileEntry(
         @Volatile
         private var closed = false
         private var closeException: Throwable? = null
+        private val closingDeferred by lazy { CompletableDeferred<Unit>() }
 
-        final override fun close(): Unit = synchronized(this) {
-            if (closed) return
+        private inline fun preClose() {
```

Comment:
no need to inline

Reply:
confusing naming


close, preClose, closeImpl, closeBlocking, closeBlockingImpl